skyportal/handlers/api/source.py

```diff
@@ -993,14 +993,34 @@ async def get(self, obj_id):
             type: string
           description: |
             datetime of observation in isoformat (e.g. 2020-12-30T12:34:10)
+        - in: query
+          name: type
+          nullable: true
+          schema:
+            type: string
+            enum: [png, pdf]
+          description: |
+            output type
+        - in: query
+          name: how_many
```

Comment:
I realize this is also an issue with the Python implementation, but `how_many` is a very generic argument name.  Makes sense for `get_nearby_offset_stars`, but not so much for `get_finding_chart` (or for this endpoint).

Perhaps use `how_many_offset_stars` or `nr_offset_stars`?

Reply:
Sure.